Sources/Sentry/SentryNetworkTracker.m

```diff
@@ -489,6 +504,10 @@ - (void)addBreadcrumbForSessionTask:(NSURLSessionTask *)sessionTask
         breadcrumbData[@"status_code"] = statusCode;
         breadcrumbData[@"reason"] =
             [NSHTTPURLResponse localizedStringForStatusCode:responseStatusCode];
+
+        if (self.isGraphQLOperationTrackingEnabled) {
+            breadcrumbData[@"operation_name"] = [sessionTask getGraphQLOperationName];
```

Comment:
`m`: On Java, they have more information, and they even add an extra breadcrumb for GraphQL. Let's prefix this with for now until we can provide more information.
```suggestion
            breadcrumbData[@"graphql_operation_name"] = [sessionTask getGraphQLOperationName];
```